File: snippets/messaging-service/send-sms/send-sms.js
// You will need twilio-node installed to run this code: https://www.npmjs.com/package/twilio

const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
const authToken = 'your_auth_token';
const client = require('twilio')(accountSid, authToken);

client.messages
      .create({
         body: 'Phantom Menace was clearly the best of the prequel trilogy.',
         messagingServiceSid: 'MGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
         to: '+15558675310'
       })
      .then(message => console.log(message.sid))
      .done();

File: snippets/messaging-service/send-sms/send-sms.cs
const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
const string authToken = "your_auth_token";

TwilioClient.Init(accountSid, authToken);

var message = MessageResource.Create(
    messagingServiceSid: "MGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
    to: new Twilio.Types.PhoneNumber("+15558675310"),
    body: "Phantom Menace was clearly the best of the prequel trilogy."
);
Console.WriteLine(message.Sid);